primefaces-selenium/primefaces-selenium-core/src/main/java/org/primefaces/selenium/PrimeSelenium.java

```diff
@@ -169,7 +179,16 @@ public static String getUrl(AbstractPrimePage page) {
      * @return the full URL
      */
     public static String getUrl(String url) {
-        return getBaseUrl() + url;
+        String baseUrl = getBaseUrl();
+        if (baseUrl == null) {
+            StringBuilder messageBuilder = new StringBuilder("Cannot determine base url. Please either configure ").append(DEPLOYMENT_BASEURL).append(" or ");
```

Comment:
please use just normal string here for readability, no StringBuilder

will be optimized by the JVM anyway

Reply:
Done